main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
Sure, so lets have that discussion as a group and if we make a decision, we can write it in the spec and everyone will then know to code that way.  In the meantime, I think we should stick to the spec.